polyglot/piranha/src/tests/mod.rs

```diff
@@ -34,14 +36,37 @@ fn initialize() {
   });
 }
 
-fn run_test(relative_path_to_tests: &str, n_files_changed: usize) {
+// Runs a piranha over the target `<relative_path_to_tests>/input` (using configurations `<relative_path_to_tests>/configuration`) 
+// and checks if the number of matches == `number_of_matches`. 
+fn run_match_test(relative_path_to_tests: &str, number_of_matches: usize) {
   let path_to_test_ff = format!("test-resources/{relative_path_to_tests}");
 
   let args = PiranhaArguments::new(CommandLineArguments {
     path_to_codebase: format!("{path_to_test_ff}/input/"),
     path_to_configurations: format!("{path_to_test_ff}/configurations/"),
   });
-  let updated_files = execute_piranha(&args);
+  let (_updated_files, matches,_rewrites) = execute_piranha(&args);
+  print!("{:?}", matches);
+  assert_eq!(matches.values().flat_map(|x|x.iter()).count(), number_of_matches);
+}
+
+// Runs a piranha over the target `<relative_path_to_tests>/input` (using configurations `<relative_path_to_tests>/configuration`) 
+// and checks if the output of piranha is same as `<relative_path_to_tests>/expected`. 
+// It also asserts the number of changed files in the expected output.
+fn run_rewrite_test(relative_path_to_tests: &str, n_files_changed: usize) {
+  let path_to_test_ff = format!("test-resources/{relative_path_to_tests}");
+
+  let args = PiranhaArguments::new(CommandLineArguments {
+    path_to_codebase: format!("{path_to_test_ff}/input/"),
+    path_to_configurations: format!("{path_to_test_ff}/configurations/"),
+  });
+  let (updated_files, _matches, rewrites) = execute_piranha(&args);
+  // Checks if there are any rewrites performed for the file
+  assert!(rewrites.values().flat_map(|x|x.iter()).count() > 0);
+
+  for r in rewrites.values().flat_map(|x|x.iter()).collect_vec() {
+    print!("{:?}", r);
```

Comment:
Same question as above about (debug?) prints.

Reply:
Yes. Thank you for pointing out.